homeassistant/data_entry_flow.py

```diff
@@ -296,9 +296,10 @@ def async_create_entry(
         data: Dict,
         description: Optional[str] = None,
         description_placeholders: Optional[Dict] = None,
+        options: Optional[dict] = None,
```

Comment:
Data Entry Flow is used for more than just config flow, for example auth. By adding this, the abstraction is getting morphed to the needs of the config entry. I don't think that we should do that. Is there another way we could achieve this?

Reply:
i could overload the async_create_entry function in config_flow, and just add the parameter in that instance type?